hack/build-manifests.sh

```diff
@@ -14,6 +14,8 @@ SKIP_TASKS="generate-odcs-compose provision-env-with-ephemeral-namespace verify-
 
 SKIP_PIPELINES="gitops-pull-request-rhtap"
 
+warning_message="# WARNING: This is an auto generated file to be used in CI, User should not modify this file directly\n# It is not used when generating tekton bundles, but users can point to it using git resolvers"
```

Comment:
> It is not used when generating tekton bundles

It isn't? I thought the build script _would_ use the committed kustomize-generated files :thinking:

Reply:
my bad, yes it is consider as [normal task](https://github.com/konflux-ci/build-definitions/blob/89d305f3996fe3fd46d1ebae6801234de41deb43/hack/build-and-push.sh#L458) and used while building the bundles, will update the comment